app/controllers/katello/api/v2/content_views_controller.rb

```diff
@@ -93,21 +93,15 @@ def show
     param :id, :identifier, :desc => N_("content view numeric identifier"), :required => true
     param :name, String, :desc => N_("module name to restrict modules for"), :required => false
     def available_puppet_modules
-      current_ids = @view.content_view_puppet_modules.map(&:uuid).reject { |p| p.nil? }
+      current_uuids = @view.content_view_puppet_modules.map(&:uuid).reject { |p| p.nil? }
```

Comment:
`@view.content_view_puppet_modules.where("uuid is not NULL").pluck(:uuid)`  would be better